### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,19 +3,30 @@
...
-# ASPxSpreadsheet - How to save and load documents from a database
+This example demonstrates how to configure the [Spreadsheet control](https://docs.devexpress.com/AspNet/16157/components/spreadsheet) to work with a database. 
+
+![Connect Spreadsheet to Database](save-to-database.gif)
+
+## Overview
+
+During page initialization, load a document from a database as a byte array and pass it to the control's [Open](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Open(System.String-DevExpress.Spreadsheet.DocumentFormat-System.Func-System.Byte---)) method to open the document in the Spreadsheet.
 
+The [Saving](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Saving) event occurs when a user clicks the built-in **Save** or **SaveAs** command on the Spreadsheet's ribbon. To save changes, handle the `Saving` event and update the data source. To save the document after a user clicks an external button, update the database in the button's [Click](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxButton.Click?p=netframework) event handler.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: First..

> when a user clicks the built-in **Save** or **SaveAs** command on the Spreadsheet's **ribbon**.

It's more precise to say as follows:

> when a user clicks the built-in **Save** or **SaveAs** command in the Spreadsheet's Ribbon UI.

Second...

> To save the document after a user clicks an external button

Of what "external" button do we speak here? Do you mean a custom button that is not a part of the (automatically generated) Ribbon UI?

Therefore, my take on this paragraph is the following:

```suggestion
The [Saving](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Saving) event occurs when a user clicks the built-in **Save** or **SaveAs** command in the Spreadsheet's Ribbon UI. To save changes, handle the `Saving` event and update the data source. To save the document after a user clicks a custom button, update the database in the button's [Click](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxButton.Click?p=netframework) event handler.
```

File: Readme.md [hunk 2]
```diff
@@ -3,19 +3,30 @@
...
+
+> **Note**  
+> When a user modifies a cell, the Spreadsheet control applies the entered value only after the user moves focus to another cell. A cell value cannot be saved until the control applies it. Call the [ApplyCellEdit](https://docs.devexpress.com/AspNet/js-ASPxClientSpreadsheet.ApplyCellEdit?p=netframework) method to force submission of the last entered value.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > When a user **modifies** a cell, the Spreadsheet control applies the entered value only after the user moves focus to another cell.

There is more than one way to modify a cell, you know. By "modified" you implied that a user changes a cell value. However, a cell formatting change, like a background color change can also be interpreted as "modification". In other words, the verb "modifies" sounds too vague in this context.

In truth, most likely, any cell modification is treated as the change you mention, not just a cell value change. Feel free to adjust this paragraph according to the facts.

> A cell value cannot be saved until the control applies it.

Again, you imply things -- a cell value could have already been saved. What value cannot be saved until the control applies it? A newly entered cell value -- a new cell value. Do not imply -- say directly, for we create documentation here.

> to force submission of the last entered value.

It sounds too complicated for such a simple concept. Moreover, we should create a better connection to the previous sentence making this paragraph a single cohesive whole.

Therefore, I suggest the following changes in this paragraph:

```suggestion
> When a user changes a cell value, the Spreadsheet control applies the entered value only after the user moves focus to another cell. A new cell value cannot be saved until the control applies it. To apply the pending cell value change, call the [ApplyCellEdit](https://docs.devexpress.com/AspNet/js-ASPxClientSpreadsheet.ApplyCellEdit?p=netframework) method.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
This is a small and correctly written topic. However, you should pay more attention to precision of your wordings.

File: Readme.md [hunk 3]
```diff
@@ -3,19 +3,30 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T190812)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Spreadsheet for ASP.NET Web Forms - How to save/load documents from/to a database
```
Comment (albertov05) [inline Readme.md]:
```suggestion
# Spreadsheet for ASP.NET Web Forms - How to save/load documents to/from a database
```

**Pull request merged**